apps/sample-app/src/components/Emails.tsx

```diff
@@ -20,25 +21,27 @@ export const EmailList: React.FC<EmailProps> = (props: EmailProps) => {
       key: m.id,
       subject: m.subject || '',
       sender: m.sender?.emailAddress?.name || '',
+      importance: 'Importance: ' + m.importance || ' ',
     };
   });
-
   const EmailExample: React.FunctionComponent = () => {
     return (
-      <div>
-        {emailItems.map(emailItem => (
-          <Text as="span" key={emailItem.key}>
-            <button key={emailItem.key}>
-              <div>
-                <DocumentCard key={emailItem.key} onClick={() => handleOpenMailItem(emailItem)}>
-                  <DocumentCardTitle title={emailItem.subject || ''} shouldTruncate />
-                  <DocumentCardTitle title={emailItem.sender || ''} shouldTruncate showAsSecondaryTitle />
-                </DocumentCard>
-              </div>
-            </button>
-          </Text>
-        ))}
-      </div>
+      <>
+        <Title3 className="paddingClass"> Recent Emails</Title3>
+        <div className="column">
+          {emailItems.map(emailItem => (
+            <Text as="span" key={emailItem.key}>
+              <DocumentCard key={emailItem.key} onClick={() => handleOpenMailItem(emailItem)}>
+                <DocumentCardTitle title={emailItem.subject || ''} shouldTruncate />
+                <DocumentCardActivity
+                  activity={emailItem.importance || ''}
+                  people={[{ name: emailItem.sender || ' ', profileImageSrc: '' }]}
```

Comment:
I've noticed some ' 's sprinkled in among ''s, maybe make them all '' for consistency? (Unless you specifically need them to be the ' ')